.github/workflows/all_tests.yml

```diff
@@ -283,6 +284,22 @@ jobs:
           shardIndex: ${{ matrix.shardIndex }}
           shardTotal: ${{ matrix.shardTotal }}
 
+  run_omadataoauth2_e2e_tests:
```

Comment:
Tämän voisi ehkä lisätä omadataoauth2samplen test workflowlle tämän workflown sijasta

Reply:
Koitan. Minusta kuitenkin näyttää, että näin tekemällä GA ei lähde ajamaan rinnakkain Kosken muista testejä, ennenkuin tämä eri workflowhun siirretty testi on valmis. Ainakin näkymä näyttää siltä:

![image](https://github.com/user-attachments/assets/7ac64f0c-5fc3-4290-acc5-a18d9801c9d0)

Reply:
tämä on pitkä testi, joka kääntää Kosken yms., joten olisi tarpeen ajaa rinnakkain muiden Kosken pitkien testien kanssa

Reply:
hmm, lähti se nyt parin minuutin jälkeen, ehkä tuo näkymän päivitys vaan oli hidas

Reply:
Tuo on se yksittäisen workflown näkymä niin kosken kaikki testit eivät näy tuossa